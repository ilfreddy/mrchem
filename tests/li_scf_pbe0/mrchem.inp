# vim:syntax=sh:

rel_prec = 1.0e-3                 # Final precision in energy

MRA {
  min_scale = -4                  # Root scale defines world size 2^{-n}
  corner = [ -1, -1, -1 ]         # Translation of corner root box [x,y,z]
  boxes = [ 2, 2, 2 ]             # Number of root boxes [x,y,z]
}

Molecule {
multiplicity = 2
$coords
Li   0.0     0.0      0.0
$end
}

WaveFunction {
  method = DFT                    # Wave function method (HF or DFT)
  restricted = false
}

DFT {
  exact_exchange = 0.25
$functionals
PBEX    0.75
PBEC    1.00
$end
}

Properties {
  scf_energy = true               # Compute ground state energy
}

SCF {
  kain = 3                        # Length of KAIN iterative history
  max_iter = 5
  orbital_thrs = 2.0e-2
  initial_guess = GTO             # Type of initial guess (none, gto, mw)
}
