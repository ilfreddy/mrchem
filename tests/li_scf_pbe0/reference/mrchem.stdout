


************************************************************
***     __  __ ____   ____ _                             ***
***    |  \/  |  _ \ / ___| |__   ___ _ __ ___           ***
***    | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \          ***
***    | |  | |  _ <| |___| | | |  __/ | | | | |         ***
***    |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|         ***
***                                                      ***
***    VERSION 0.0.1 (rev. 839f67b)                      ***
***                                                      ***
***    Stig Rune Jensen <stig.r.jensen@uit.no>           ***
***    Luca Frediani    <luca.frediani@uit.no>           ***
***    Peter Wind       <peter.wind@uit.no>              ***
***                                                      ***
************************************************************


+++ Serial execution


------------------------------------------------------------

 MRCPP version   : X.Y.Z
 Git revision    : 735162b

 Linear algebra  : EIGEN
 Parallelization : NONE

------------------------------------------------------------




============================================================
                  MultiResolution Analysis                  
------------------------------------------------------------
 polynomial order = 5
 polynomial type  = Interpolating
------------------------------------------------------------
 unit length      = 16.000000
 total boxes      = 8
 boxes            = [           2           2           2 ]
 lower bounds     = [  -16.000000  -16.000000  -16.000000 ]
 upper bounds     = [   16.000000   16.000000   16.000000 ]
 total length     = [   32.000000   32.000000   32.000000 ]
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     Li          0.00000e+00   0.00000e+00   0.00000e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     Li           3.00000e+00   1.00000e-03   2.61594e-03
============================================================


============================================================
                Setting up Gaussian-type MOs
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    a    1                       9.999999988178122e-01
    1    a    1                       1.000000048121801e+00
------------------------------------------------------------
                 Wall time: 1.39542e-01 sec 
============================================================


============================================================
                Setting up Gaussian-type MOs
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    b    1                       9.999999988146172e-01
------------------------------------------------------------
                 Wall time: 8.06946e-02 sec 
============================================================


============================================================
 OrbitalVector:   3 orbitals     3 occupied     3 electrons
------------------------------------------------------------
   n  RankID           Norm          Spin Occ      Error    
------------------------------------------------------------
   0     0       9.999999988178e-01    a   1    1.00000e+00
   1     0       1.000000048122e+00    a   1    1.00000e+00
   2     0       9.999999988146e-01    b   1    1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     840      1.68024e-01
 Coulomb density                       392      1.75713e-01
 Coulomb potential                     200      1.19068e+00
 XC alpha density                      392      1.22780e-01
 XC beta density                       392      6.17939e-02
 XC evaluate xcfun                    3528      2.93169e+00
 XC energy                             392      3.20050e-05
 XC potential                          784      8.46727e-02
 XC grid size                          392           594432
 XC grid change                       -384          -580608
 Hartree-Fock exchange                1112      4.75441e+00
------------------------------------------------------------
                 Wall time: 9.56111e+00 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         3.041413120000000e-01
 Potential part                       7.264257380000000e-01
------------------------------------------------------------
                 Wall time: 1.03063e+00 sec 
============================================================




####################### SCF cycle  1 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              2.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        2.43209e-03
 Diagonalizing matrix                           9.52860e-05
 Rotating orbitals                              2.96442e-02
------------------------------------------------------------
                 Wall time: 3.22177e-02 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       0.000000000000000e+00
 Electronic energy                   -7.413819949580084e+00
------------------------------------------------------------
 Total energy                        -7.413819949580084e+00
------------------------------------------------------------
                 Wall time: 7.33498e-01 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -2.02627e+00
 Creating operator with lambda                 -1.39733e-01
 Creating operator with lambda                 -2.01602e+00
------------------------------------------------------------
                 Wall time: 3.00465e-01 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 7.21511e-01
 Matrix part                                    3.32430e-05
------------------------------------------------------------
   0       1.49685e+00    584            0.00000e+00      0
   1       2.67592e-01    712            0.00000e+00      0
   2       1.49693e+00    392            0.00000e+00      0
------------------------------------------------------------
                 Wall time: 7.45569e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.00369e+00   328    0.00000e+00     0   2.18941e+00
  1    9.97845e-01   264    0.00000e+00     0   2.01578e+00
  2    1.00480e+00   328    0.00000e+00     0   2.03153e+00
------------------------------------------------------------
                 Wall time: 6.23679e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             2.63500e-05
 Copy orbitals                                  9.92021e-03
 Copy orbital updates                           1.22218e-02
------------------------------------------------------------
                 Wall time: 2.22244e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     840      1.70943e-01
 Coulomb density                       392      1.73763e-01
 Coulomb potential                     200      1.24221e+00
 XC alpha density                      392      1.22948e-01
 XC beta density                       392      6.43313e-02
 XC evaluate xcfun                    3528      2.98394e+00
 XC energy                             392      2.99710e-05
 XC potential                          784      8.57098e-02
 XC grid size                          392           594432
 XC grid change                       -384          -580608
 Hartree-Fock exchange                1176      4.79110e+00
------------------------------------------------------------
                 Wall time: 9.70580e+00 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         3.118754690000000e-01
 Potential part                       6.954999930000000e-01
------------------------------------------------------------
                 Wall time: 1.00744e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.06049e+00   3.76631e-02        328    a    1    0
  1  -1.41630e-01   4.43976e-02        264    a    1    0
  2  -2.05059e+00   3.85938e-02        328    b    1    0
------------------------------------------------------------
 Total error:                            6.98509e-02    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -4.182018094178742e+00    -4.18202e+00    1
 Kinetic       7.370996345565676e+00     7.37100e+00    1
 N-E          -1.703502983029751e+01    -1.70350e+01    1
 Coulomb       4.061901604356890e+00     4.06190e+00    1
 Exchange     -4.448032885822927e-01    -4.44803e-01    1
 X-C          -1.366884780622849e+00    -1.36688e+00    1
------------------------------------------------------------
 Electronic   -7.413819949580084e+00    -7.41382e+00    1
 Nuclear       0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Total        -7.413819949580084e+00    -7.41382e+00    1
============================================================


################ Wall time: 1.89249e+01 sec ################




####################### SCF cycle  2 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              2.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        3.20325e-03
 Diagonalizing matrix                           8.70480e-05
 Rotating orbitals                              2.92612e-02
------------------------------------------------------------
                 Wall time: 3.26067e-02 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       0.000000000000000e+00
 Electronic energy                   -7.465195706886618e+00
------------------------------------------------------------
 Total energy                        -7.465195706886618e+00
------------------------------------------------------------
                 Wall time: 7.30509e-01 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -2.06057e+00
 Creating operator with lambda                 -1.41556e-01
 Creating operator with lambda                 -2.05059e+00
------------------------------------------------------------
                 Wall time: 2.93981e-01 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 6.98905e-01
 Matrix part                                    3.39740e-05
------------------------------------------------------------
   0       1.53579e+00    392            0.00000e+00      0
   1       2.58188e-01    712            0.00000e+00      0
   2       1.53349e+00    392            0.00000e+00      0
------------------------------------------------------------
                 Wall time: 7.21523e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    9.97094e-01   328    0.00000e+00     0   2.09436e+00
  1    9.96666e-01   264    0.00000e+00     0   2.06646e+00
  2    9.97294e-01   328    0.00000e+00     0   2.02808e+00
------------------------------------------------------------
                 Wall time: 6.18899e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.30220e-05
 Copy orbitals                                  9.48876e-03
 Copy orbital updates                           1.34187e-02
------------------------------------------------------------
                 Wall time: 2.29631e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     840      1.63075e-01
 Coulomb density                       392      1.62290e-01
 Coulomb potential                     200      1.22032e+00
 XC alpha density                      392      1.13785e-01
 XC beta density                       392      6.21313e-02
 XC evaluate xcfun                    3528      2.94727e+00
 XC energy                             392      3.04240e-05
 XC potential                          784      9.14964e-02
 XC grid size                          392           594432
 XC grid change                       -384          -580608
 Hartree-Fock exchange                1176      4.57560e+00
------------------------------------------------------------
                 Wall time: 9.40361e+00 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         3.065237880000000e-01
 Potential part                       7.069505130000000e-01
------------------------------------------------------------
                 Wall time: 1.01354e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.05590e+00   3.74540e-03        328    a    1    1
  1  -1.41007e-01   1.67366e-02        264    a    1    1
  2  -2.04567e+00   3.66210e-03        328    b    1    1
------------------------------------------------------------
 Total error:                            1.75372e-02    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -4.252714179175324e+00    -7.06961e-02    1
 Kinetic       7.311713966078253e+00    -5.92824e-02    1
 N-E          -1.701188463939257e+01     2.31452e-02    1
 Coulomb       4.035279564821574e+00    -2.66220e-02    1
 Exchange     -4.414137204317500e-01     3.38957e-03    1
 X-C          -1.358890877962122e+00     7.99390e-03    1
------------------------------------------------------------
 Electronic   -7.465195706886618e+00    -5.13758e-02    1
 Nuclear       0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Total        -7.465195706886618e+00    -5.13758e-02    1
============================================================


################ Wall time: 1.85486e+01 sec ################


============================================================
                      Convergence rate
------------------------------------------------------------
Iter    OrbError       Property                   Update  
------------------------------------------------------------
  1    6.98509e-02    -7.413819949580084e+00   -7.41382e+00
  2    1.75372e-02    -7.465195706886618e+00   -5.13758e-02
------------------------------------------------------------
                      SCF converged!!!                      
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        2.43713e-03
 Diagonalizing matrix                           9.19060e-05
 Rotating orbitals                              3.69080e-02
------------------------------------------------------------
                 Wall time: 3.94909e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     840      1.61998e-01
 Coulomb density                       392      1.65899e-01
 Coulomb potential                     200      1.19182e+00
 XC alpha density                      392      1.10618e-01
 XC beta density                       392      6.08600e-02
 XC evaluate xcfun                    3528      2.97055e+00
 XC energy                             392      3.09770e-05
 XC potential                          784      8.98672e-02
 XC grid size                          392           594432
 XC grid change                       -384          -580608
 Hartree-Fock exchange                1176      4.69995e+00
------------------------------------------------------------
                 Wall time: 9.51829e+00 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 7.89249e-01 sec 
============================================================


============================================================
                        Fock matrix
------------------------------------------------------------
-2.05590e+00  0.00000e+00  0.00000e+00
 0.00000e+00 -1.41007e-01  0.00000e+00
 0.00000e+00  0.00000e+00 -2.04567e+00
============================================================


============================================================
                      Orbital energies
------------------------------------------------------------
    n  spin  occ                            epsilon  
------------------------------------------------------------
    0    a    1                      -2.055896857655864e+00
    1    a    1                      -2.045667932171003e+00
    2    b    1                      -1.410070339564735e-01
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     Li          0.00000e+00   0.00000e+00   0.00000e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


                                                            
============================================================
                         SCF Energy                         
------------------------------------------------------------
                                                            
 Sum orbital energy:                 -4.242571823783341e+00
 Kinetic energy:                      7.380423976192780e+00
 E-N energy:                         -1.708869919300545e+01
 Coulomb energy:                      4.048561381710947e+00
 Exchange energy:                    -4.427613404288200e-01
 X-C energy:                         -1.363562241663444e+00
 El. External field energy:           0.000000000000000e+00
 Nuc. External field energy:          0.000000000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Electronic energy:                  -7.466037417193983e+00
 Nuclear energy:                      0.000000000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Total energy       (au)             -7.466037417193983e+00
                    (kJ/mol)         -1.960207840174859e+04
                    (kcal/mol)       -4.685009175197528e+03
                    (eV)             -2.031612189569730e+02
                                                            
============================================================
                                                            



************************************************************
***                                                      ***
***                    Exiting MRChem                    ***
***                                                      ***
***               Wall time: 5.876621e+01                ***
***                                                      ***
************************************************************


